VERSIONING.md

```diff
@@ -0,0 +1,113 @@
+# Versioning details
+
+This document describes the versioning and stability policy of components
+shipped from this repository, as per the [OpenTelemetry versioning and stability
+specification](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/versioning-and-stability.md).
+
+This repo follows [SemVer V2](https://semver.org/spec/v2.0.0.html) guidelines.
+This means that, for any stable packages released from this repo, all public
+APIs will remain [backward
+compatible](https://github.com/dotnet/runtime/blob/master/docs/coding-guidelines/breaking-change-rules.md#breaking-change-rules),
+unless a major version bump occurs. This applies to the API, SDK, as well as
+Exporters, Instrumentation etc. shipped from this repo.
+
+For example, users can take a dependency on 1.0.0 version of any package, with
+the assurance that all future releases until 2.0.0 will be backward compatible.
+Any method which is planned to be removed in 2.0, will be marked
+[Obsolete](https://docs.microsoft.com/dotnet/api/system.obsoleteattribute)
+first.
+
+## API and SDK compatibility
+
+API packages are supported with any SDK version that has same MAJOR version and
+equal or greater MINOR or PATCH version. For example, application/library that is instrumented
+with OpenTelemetry.API 1.0, will be compatible with SDK version 1.0 or higher.
+
+
```

Comment:
nit: remove extra white lines.

Reply:
I have to update the contents of this section to clarify the tighter version requirement between api/sdk and sdk/exporters from this repo.